### PR TITLE
Show dashboards for several projects

### PR DESCRIPTION
**Pull request by youknowriad**
This PR allows the UI to show dashboards for multiple projects using the `/project/{id}` url. Ideally we'd use "slugs" but there's no slugs in the database right now, let's introduce them later.

For example:

 - Gutenberg dashboard: https://codehealth-git-add-multi-project-ui-riad.vercel.app/project/1 
 - WordPress dashboard: https://codehealth-git-add-multi-project-ui-riad.vercel.app/project/2

### CODE REVIEW

File: components/header/index.js [hunk 1]
```diff
@@ -58,13 +62,28 @@ function UserMenu() {
...
 }
 
+function ProjectName({ id }) {
+  const { data: project } = useSWR('/api/project/' + id, fetcher)
+  return project?.name ?? null
+}
+
 export default function Header() {
+  const router = useRouter()
+  const { project_id } = router.query
```
Comment (ntsekouras) [inline components/header/index.js:72]: Nit: I guess you could move this inside `ProjectName` and even inline the `fetcher` method.

File: pages/api/projects.js [hunk 2]
```diff
@@ -1,21 +1,10 @@
 import { PSDB } from 'planetscale-node'
-import { getSession } from 'next-auth/react'
 
 const conn = new PSDB('main')
 
 export default async (req, res) => {
-  const session = await getSession({ req })
```
Comment (ntsekouras) [inline pages/api/projects.js:7]: This wasn't used?

**Review by ntsekouras — COMMENTED** [review]
Thanks Riad!

**Comment by ntsekouras** [comment]
Great timing! I was approving, but it seems you merged in the same time and turned my approval to comment :)

File: pages/api/projects.js (hunk 2 above)
Comment (youknowriad) [reply pages/api/projects.js:7]: I changed the purpose of the API. Per user projects is something that I'll add later if I have time :)

File: components/header/index.js (hunk 1 above)
Comment (youknowriad) [reply components/header/index.js:72]: no, I can't because it's not synchronous, I prefer to only render. `ProjectName` when the id is defined.

**Pull request merged**